changes/2374.fixed

```diff
@@ -0,0 +1,4 @@
+Revised documentation for recommended parameters to use when running `nautobot-server dumpdata`.
+Revised documentation around preparing to run `nautobot-server loaddata`.
+Added documentation to run `nautobot-server trace_paths` after `nautobot-server loaddata`.
```

Comment:
Potentially move to `2374.added` or reword to `Fixed` or `Revised`?

Reply:
I mean it's an *addition* that *fixed* an omission in the docs. :)